policyengine_api/endpoints/economy/economy.py

```diff
@@ -219,8 +219,6 @@ def get_economic_impact(
         )
         return dict(
             status=result["status"],
```

Comment:
```suggestion
            status=result["status"],
            queue_position=queue_pos,
```
We do actually still want the queue position, just want to remove the redundant message that also gives the queue position